examples/puppet_headnode/manifests/static/boot_notify.py

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python
+import re
+import os
+
+# !/usr/bin/env python
```

Comment:
Also, there cannot be a space between the `#` and the `!`.

This is obviously untenable, this is actually strictly incorrect code. Furthermore, we probably can't use `# noqa` here, since the shell is going to be looking at it and it imposes some restrictions on what that argument can look like.

I'm assuming the tool complains if you put it back where it was?

Since this is an example, one option is to just remove it entirely.